tests/config_test.go

```diff
@@ -40,6 +40,7 @@ import (
 
 	"kubevirt.io/kubevirt/pkg/config"
 	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/tests/libclient"
```

Comment:
nit. on most of the files put the import next to the import of kubevirt/tests

Reply:
https://github.com/kubevirt/kubevirt/blob/main/docs/coding-conventions.md#uniform-import-order-and-naming
I agree with @ShellyKa13

Reply:
ahh I wasn't reading it right. Will fix.

Reply:
@RamLavi yes, I think we need to add this enhancement in the coding convention :)